### PR TITLE
Don't transpile ES Classes.

### LINKED ISSUE #13: ES classes being transpiled
This is breaking web components.

See https://github.com/GoogleChromeLabs/squoosh/tree/components-merge where I've tried to merge components in.

### PR DESCRIPTION
**Pull request by developit**
This should fix the WC issues. I've verified that the classes are left in-tact in the compiled output.

Fixes #13.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+        },
+        "exclude": [
+          "transform-regenerator",
+          "transform-es2015-typeof-symbol"
+        ]
+      }
+    ]
+  ],
+  "plugins": [
+    "transform-decorators-legacy",
```
[the remark this answers is not in the document]
Comment (developit) [reply .babelrc]: I don't think so, at least I hope not. I'll remove this (and the env preset).

File: .babelrc [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
+        "modules": false,
+        "targets": {
+          "browsers": [
+            "last 2 versions",
+            "not android>0",
+            "not ie>0"
+          ]
+        },
+        "exclude": [
+          "transform-regenerator",
```
[the remark this answers is not in the document]
Comment (developit) [reply .babelrc]: Not sure, but when I remove `env` we definitely won't need them.

**Comment by developit** [comment]
Alright, updated based on Jake's suggestion to ditch `env` entirely.  Now we're only applying babel to deal with JSX, class properties and PropTypes.

**Comment by mathiasbynens** [comment]
Can I read more about the decision to ditch `env` somewhere? I’m especially interested as this goes against our external messaging.

**Comment by developit** [comment]
@mathiasbynens I think it was just based on our browser support requirements being fairly relaxed.

**Pull request merged**